oke/README.md

```diff
@@ -2,22 +2,40 @@
 
 ## Overview
 
-Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE) is a fully-managed Kubernetes service for deploying and running your containerized applications on Oracle Cloud. Datadog provides you with comprehensive visibility to your Kubernetes clusters managed by OKE. Once you have enabled your Datadog integration, you can view your Kubernetes infrastructure, monitor live processes, and track key metrics from all of your pods and containers in one place.
+Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE) is a managed Kubernetes service that simplifies the operations of enterprise-grade Kubernetes at scale. 
+
+This integration collects metrics and tags from the [`oci_oke`][1] namespace to help you monitor your Kubernetes control plane, clusters, and node states. 
+
+Deploying the [Datadog Agent][2] on your OKE cluster can also help you track the load on your clusters, pods, and individual nodes to get better insights into how to provision and deploy your resources.
+
+In addition to monitoring your nodes, pods, and containers, the Agent can also collect and report metrics from the services running in your cluster, so that you can:
+
+- Explore your OKE clusters with [pre-conifgured Kubernetes dashboards][3]
+- Monitor containers and processes in real time
+- Automatically track and monitor containerized services
 
 ## Setup
 
+Once you set up the [Oracle Cloud Infrastructure][4] integration, ensure that the `oci_oke` namespace is included in your [Connector Hub][5].
+
 Because Datadog already integrates with Kubernetes, it is ready-made to monitor OKE. If you're running the Agent in a Kubernetes cluster and plan to migrate to OKE, you can continue monitoring your cluster with Datadog.
 
-Additionally, OKE node pools are supported.
+Deploying the Agent as a DaemonSet via our [Helm chart][6] is the most straightforward (and recommended) method, since it ensures that the Agent will run as a pod on every node within your cluster and that each new node automatically has the Agent installed. You can also configure the Agent to collect process data, traces, and logs by adding a few extra lines to a Helm values file. Additionally, OKE node pools are supported.
```

Comment:
```suggestion
Deploying the Agent as a DaemonSet with the [Helm chart][6] is the most straightforward (and recommended) method, since it ensures that the Agent will run as a pod on every node within your cluster and that each new node automatically has the Agent installed. You can also configure the Agent to collect process data, traces, and logs by adding a few extra lines to a Helm values file. Additionally, OKE node pools are supported.
```